src/main/c/ssl.c

```diff
@@ -2036,6 +2036,17 @@ TCN_IMPLEMENT_CALL(void, SSL, setState)(TCN_STDARGS,
     SSL_set_state(ssl_, state);
 }
 
+TCN_IMPLEMENT_CALL(jint, SSL, release0)(TCN_STDARGS)
```

Comment:
No sry...what I gave will just print something that identifies 1.0.2.  What I was thinking of would be represented by our current `versionString` method (compile time version string).

I was originally assuming this was just for informational purposes ... but we actually plan on varying behavior (in java code?) depending on the compile time library?